index.html

```diff
@@ -61,5 +61,203 @@ <h1>
         </li>
       </ul>
     </section>
+    
+    <section class="work-section">
```

Comment:
- [ ] Great job so far 👍 your design looks good and it follows the Figma template. You'll notice that the bottom-left of the first page is curved and that the background color changes as you leave the first page. It would be great if you could implement this functionality 💯 
- Your design:
 ![Capture1](https://user-images.githubusercontent.com/85113567/154784092-90cd6697-52c2-4ba0-bbc0-588b6e82431b.PNG)
- Figma design:
![Capture4](https://user-images.githubusercontent.com/85113567/154784056-a2b8e4fa-a17c-4301-a98b-1fd975092291.PNG)